src/nbl/asset/interchange/CImageLoaderJPG.cpp

```diff
@@ -147,7 +147,7 @@ namespace jpeg
 #endif // _NBL_COMPILE_WITH_LIBJPEG_
 
 //! returns true if the file maybe is able to be loaded by this class
-bool CImageLoaderJPG::isALoadableFileFormat(io::IReadFile* _file) const
+bool CImageLoaderJPG::isALoadableFileFormat(system::IFile* _file) const
```

Comment:
in the asset `.cpp`s you can just do `using namespace nbl::system;` to save yourself some writin (but obvs not in headers)